backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/mapper/UpdateModResponseMapper.java

```diff
@@ -0,0 +1,27 @@
+package de.bund.digitalservice.ris.norms.adapter.input.restapi.mapper;
+
+import de.bund.digitalservice.ris.norms.adapter.input.restapi.schema.UpdateModResponseSchema;
+import de.bund.digitalservice.ris.norms.application.port.input.UpdateModUseCase;
+import de.bund.digitalservice.ris.norms.domain.entity.Norm;
+import org.springframework.stereotype.Service;
+
+/** Mapper class for converting to a {@link UpdateModResponseSchema}. */
+@Service
```

Comment:
I would say this is a `@Service` but rather a simple class with a static method? Like the other mappers?